requirements.txt

```diff
@@ -18,6 +17,7 @@ jinja2
 argcomplete
 jc
 sentry-sdk
-prometheus_client
-ec2_metadata
-
+prometheus-client
```

Comment:
Should we consider locking the versions so that builds are always reproducible and don't get affected by version incompatibilities?